crates/bdk/README.md

```diff
@@ -218,9 +218,11 @@ submitted for inclusion in the work by you, as defined in the Apache-2.0
 license, shall be dual licensed as above, without any additional terms or
 conditions.
 
+[`Wallet`]: https://docs.rs/bdk/1.0.0-alpha.7/bdk/wallet/struct.Wallet.html
```

Comment:
Perhaps this could also point to 'latest', only problem is that the latest on docs.rs is still on 0.29

Reply:
Once we get a final 1.0.0 we can remove the "pre-release" flag and make it the latest on crates.io and docs.rs.